content/en/blog/_posts/2023-12-05-image-filesystem.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: blog
+title: 'Image Filesystem: Configuring Kubernetes to store containers on a separate filesystem'
+date: 2023-12-05
+slug: kubernetes-separate-imagefilesystem
+---
+
+**Authors:** Kevin Hannon (Red Hat)
+
+A common issue in running/operating Kubernetes clusters is running out of disk space.
+When the node is provisioned, you should aim to have a good amount of storage space for the container runtime.  
```

Comment:
The container runtime to me, and perhaps others, would be runc or containerd and such, whereas container images are a different thing. Unless we mean "the time when the container runs" as the "container runtime". We could clarify this to remove ambiguity such that there should be enough storage to store (or save) container images before a container can be run.

Reply:
CRI includes both a runtime service and an image service. CRI implementators will pull images also and they also run containers.

So I think my statement is correct.

Reply:
```suggestion
When the node is provisioned, you should aim to have a good amount of storage space for your container images and running containers.
```

Reply:
updated.